src/app/features/todos/add-default-todo-views.ts

```diff
@@ -0,0 +1,155 @@
+    configData[viewConfigKey].config.filters = existingFilters;
+  }
+
+  // Details View
+  if (!configData[viewConfigKey + "/:id"]) {
+    configData[viewConfigKey + "/:id"] = JSON.parse(
+      JSON.stringify(defaultTodoDetailsView),
+    );
+  }
+
+  return configPart;
+};
+
+export const defaultTodoListView = {
+  component: "EntityList",
+  config: {
+    entityType: "Todo",
+    columns: [
+      "deadline",
+      "subject",
+      "assignedTo",
+      "startDate",
+      "relatedEntities",
+    ],
+    filters: [
+      { id: "assignedTo", default: PLACEHOLDERS.CURRENT_USER },
+      { id: "todo-due-status", type: "prebuilt" },
+    ],
+    defaultSort: { active: "deadline", direction: "asc" },
+    clickMode: "popup-details",
+  },
+};
+
+export const defaultTodoDetailsView = {
+  component: "EntityDetails",
+  config: {
+    entityType: Todo.ENTITY_TYPE,
+    panels: [
+      {
+        title: "Overview",
+        components: [
+          {
+            title: "",
+            component: "Form",
+            config: {
+              fieldGroups: [
+                {
+                  fields: [
+                    "subject",
+                    "deadline",
+                    "startDate",
+                    "description",
+                    "assignedTo",
+                    "relatedEntities",
+                    "repetitionInterval",
+                    "completed",
+                  ],
+                },
+              ],
+            },
+          },
+        ],
+      },
+    ],
+  },
+};
+
+/**
+ * Special filter with pre-defined categories of Todo items.
+ */
+export const todoDueStatusFilter: PrebuiltFilterConfig<Todo> = {
+  id: "todo-due-status",
+  type: "prebuilt",
+  label: $localize`Tasks due`,
+  options: [
+    {
+      key: "current",
+      label: $localize`:Filter-option for todos:Currently Active`,
+      filter: {
+        $and: [
+          { isCompleted: false },
+          {
+            $or: [
+              {
+                startDate: {
+                  $exists: false,
+                },
+              },
+              {
+                startDate: {
+                  $lte: moment().format("YYYY-MM-DD"),
+                  $gt: "",
+                },
+              },
+              {
+                deadline: {
+                  $lte: moment().format("YYYY-MM-DD"),
+                  $gt: "",
+                },
+              },
+            ],
+          },
+        ],
+      } as DataFilter<Todo>,
+    },
+    {
+      key: "overdue",
+      label: $localize`:Filter-option for todos:Overdue`,
+      filter: { isOverdue: true },
+    },
+    {
+      key: "completed",
+      label: $localize`:Filter-option for todos:Completed`,
+      filter: { isCompleted: true, isActive: { $exists: true } },
+    },
+    {
+      key: "open",
+      label: $localize`:Filter-option for todos:All Open`,
+      filter: { isCompleted: false },
+    },
+    { key: "", label: $localize`Any`, filter: {} },
```

Comment:
Currently, when selecting the "Any" option, the selected value is not displayed as expected in the dropdown-

![image](https://github.com/user-attachments/assets/7809898e-1564-4b5d-a6a3-2e38c8bf8a22)

```suggestion
    { key: "any", label: $localize`Any`, filter: {} },
```